modules/passkey/src/utils/webauthn.ts

```diff
@@ -0,0 +1,186 @@
+import { ethers } from 'ethers'
```

Comment:
I think the `package.json` needs to be updated in order to include the correct dependencies used by src. Note that dependencies used here cannot be `devDependencies` but need to be regular `dependencies`

Reply:
> I think the `package.json` needs to be updated in order to include the correct dependencies used by src. Note that dependencies used here cannot be `devDependencies` but need to be regular `dependencies`

`peerDependencies` should work too, right? https://nodejs.org/en/blog/npm/peer-dependencies

Reply:
It would... but I wanted to avoid using `ethers` if we don't need it because:
- it pulls in a lot of stuff
- v5 vs v6 shenanigans